medicines-api/overlays/prod/deployment.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: medicines-api
+spec:
+  replicas: 2
+  template:
+    spec:
+      containers:
+        - name: medicines-api
+          image: mhraproductsproductionregistry.azurecr.io/products/medicines-api
+          env:
+            - name: SEARCH_SERVICE
+              value: mhraproductsproduction
+            - name: RUST_LOG
+              value: api=debug,info
```

Comment:
I guess we might want to reduce the amount of logging to just "info" once it's been running for a while without issues?